docs/tutorials/comparison.md

```diff
@@ -79,302 +123,753 @@ source1 = "FirstSourceName"
+`{fps_den}`      | The frame rate denominator                       | `1001`
+`{width}`        | The width of the current frame in pixels         | `1920`
+`{height}`       | The height of the current frame in pixels        | `1080`
+`{format}`       | The video format                                 | `YUV420P10`
 
-**Frameinfo:** Lists the frame number, type, and source name in the top left of the videos.
+!!!
+We recommend changing this to `{frame}_{index}_{Name}`, which is a more friendly naming scheme.
+!!!
 
-**FrameProp:** Sets the source name you entered earlier, for correct labelling on slow.pics.
+!!!warning
+The `{Name}` placeholder may include extraneous data in newer versions of VSPreview (e.g. `b'sourceName'` instead of `sourceName`).
+!!!
 
-**Output:** Simply makes the clips appear in VS-Preview.
 ==-
 
-## Screenshotting
+### Comp
 
-There are 3 methods of screenshotting:
+The comp bar is primarily used for creating [automatic](#automatic-badge-variant-secondary-text-fastest-option)/[semi-automatic](#semi-automatic-badge-icon-variant-primary-text-recommended) comparisons to [Slowpoke Pics](https://slow.pics):
 
-**Full-auto:** Allowing VS-Preview to do everything for you, by far the fastest option.
+![VSPreview comp bar](/static/comparison/vsp-comp.png)
 
-**Semi-auto:** Manually marking the frame numbers as you go through the clips, then letting VS-Preview automatically extract, compress, and upload them to slow.pics. Generally the recommended option.
+==- 1 - Comparison title
 
-**Bolt-action:** Saving the images locally as you go through the clips, then uploading them manually later. Mainly useful for uploading to other sites and keeping the comparison archived locally, but also has other benefits like faster and more efficient compression.
+The title of your comparison.
 
-+++ Full-auto
+#### Naming scheme
 
-- Click the `comp` button in the bottom right of the toolbar.
-- Untick `I` so only `P` and `B` are selected, or only select `B` if you want matching frame-types.
-- Tick the `public` box to make the comparison available on the slow.pics homepage.
-- Name the collection appropriately for Slow.pics, get the TMDB ID from <https://www.themoviedb.org/> and paste it in the TMDB box, with it correctly set to Movie/TV depending on the content.
-- Set the tag to the same name as listed on TMDB.
-- Set "Delete After" to your desired value, by default comps expire after 2 years of not being viewed.
-- Under `random` enter a frame count amount, you should use a large number of images to make the comparison as effective as possible, ideally at least 40 since not all frames will be useful.
-- Make sure your PNG compression is set to 0 (Max) under Settings -> Main*
-- Hit `Start Upload` and patiently wait while VS-Preview collects the frames and uploads the comparison.
+When creating your comparison, we recommend naming it with the show title and sources used. Some examples are:
```

Comment:
should mention some reasoning like, comps should be named properly enough that you should not need outside context to understand it